doc/source/KnownIssues.rst

```diff
@@ -88,6 +88,3 @@ macOS
 5. Errors such as ``Symbol not found: __cg_png_create_info_struct``
 
    Can happen when trying to use the raster plotting scripts in ``fv3-jedi-tools``. In that case, exporting ``DYLD_LIBRARY_PATH=/usr/lib/:$DYLD_LIBRARY_PATH`` can help. If ``git`` commands fail after this, you might need to verify where ``which git`` points to (Homebrew vs module) and unload the ``git`` module.
-
```

Comment:
This is no longer true, we fixed the met 10.1.1 build (there are now issues with 11.x.y, but that's a different story and being worked on by the MET developers).